connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -117,10 +119,20 @@ public void done() {
         HeLogic.handleHeToken(parameters, showLoadingCallback, heTokenCallback, logSessionId, useStaging);
     }
 
-    private static void setButtonClickedTimestamp() {
+    private static void handleButtonClickedAnalytics() {
+        updateLogSessionIdIfTooOld();
         tsLoginButtonClicked = System.currentTimeMillis();
     }
 
+    private static void updateLogSessionIdIfTooOld() {
+        Calendar calendar = Calendar.getInstance();
+        calendar.add(Calendar.MINUTE, -10);
```

Comment:
Is it usefult to add information here about why we use 10 minutes? (Was this the one that was discussed earlier, where the minimum of other settings was 10 minutes?)

Reply:
I'll add some context :) 👍

Reply:
Better now?